pkg/ebpf/c/common/arguments.h

```diff
@@ -1,8 +1,13 @@
-#ifndef __TRACEE_EVENTS_ARGS_H__
-#define __TRACEE_EVENTS_ARGS_H__
+#ifndef __COMMON_ARGUMENTS_H__
+#define __COMMON_ARGUMENTS_H__
+
+#include <vmlinux.h>
+#include <vmlinux_flavors.h>
+#include <vmlinux_missing.h>
```

Comment:
I tried to create (just for now) a pattern where all 3 vmlinux files were included in all headers. Possibly not a good idea but I thought it could help other developers (as many tend to be confused by types, missing types and type flavors).